hbase-server/src/main/java/org/apache/hadoop/hbase/procedure2/store/region/RegionProcedureStore.java

```diff
@@ -547,6 +551,10 @@ public void insert(Procedure<?> proc, Procedure<?>[] subProcs) {
     }
     List<Mutation> mutations = new ArrayList<>(subProcs.length + 1);
     List<byte[]> rowsToLock = new ArrayList<>(subProcs.length + 1);
+    // Insert procedure may be called by master's rpc call. There are some check about the rpc call
+    // when mutate region. Here unset the current rpc call and set it back in finally block.
+    // See HBASE-23895 for more details.
+    Optional<RpcCall> rpcCall = RpcServer.unsetCurrentCall();
```

Comment:
Pity we have to write the comment three times...

Can be a follow on, to abstract a method for this logic...